app/templates/content/index.hbs

```diff
@@ -14,7 +14,7 @@
                                         {{#if author.unregisteredContributor}}
                                             <li>{{author.unregisteredContributor}}</li>
                                         {{else}}
-                                            <li><a href={{author.users.profileURL}} onclick={{action 'click' 'link' 'Content - Author' author.users.profileURL}}>{{author.users.givenName}} {{author.users.familyName}}</a></li>
+                                            <li><a href={{author.users.profileURL}} onclick={{action 'click' 'link' 'Content - Author' author.users.profileURL}}>{{author.users.fullName}}</a></li>
```

Comment:
After some pondering, I had a quick product consult and I think there is a slightly more ideal solution:

1. Add a helper to return Given + Family Name if both exist, otherwise return Full name
2. Add tests for the helper

The reasoning is that Full name can contain a multitude of other information. Some people add other things to their fullname like "Alexander J. Schiller, PhD" which adds a fair amount of unnecessary clutter compared to "Alexander Schiller." It is safer to keep everyone the same and only use the subpar Full name when no given/family name exist. 

Let me know if you have any questions about that.